### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by camper2018**

### CODE REVIEW

File: database/fakeData.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+// Condition: String,
```
Comment (creationator) [inline database/fakeData.js]: You probably can remove these commented out lines. Your schema will define what the types are right?

File: database/index.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  Sport: String,
+  MPN: Number,
+  ArticleType: String,
+  OuterSole: String,
+  Activity: String
+}, {strict: false});
+itemSpecificsSchema.set('validateBeforeSave', false);
+const itemSpecifics = mongoose.model('itemSpecifics', itemSpecificsSchema);
+
+const save = (array) => {
```
[the remark this answers is not in the document]
Comment (manuelreneduran) [reply database/index.js]: I'm specifically talking about line 32, the save function.

File: database/fakeData.js (hunk 1 above)
Comment (camper2018) [reply database/fakeData.js]: sure I will do that . Thanks

File: database/index.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (camper2018) [reply database/index.js]: Yes it is saving entire array of 100 objects. I think it runs only one time where I am invoking save function on line 39.

[the remark this answers is not in the document]
Comment (camper2018) [reply database/index.js]: > I'm specifically talking about line 32, the save function.

Yes it is saving entire array of 100 objects. I think it runs only one time where I am invoking save function on line 39.

**Pull request merged**